api/apps/articles/model/index.js

```diff
@@ -81,9 +81,7 @@ export const save = (input, accessToken, options, callback) => {
 
         // Eventually convert these to Joi custom extensions
         modifiedArticle.updated_at = new Date()
-        if (input.hero_section && input.hero_section.type === 'fullscreen') {
-          modifiedArticle.title = modifiedArticle.hero_section.title
-        }
+
         if (input.author) { modifiedArticle.author = input.author }
```

Comment:
Conditionals are always a bit easier to read at a glance if broken out like 
```js
if (input.author) { 
  modifiedArticle.author = input.author 
}
```
Also (though i'm not suggesting one does this as it typically leads to subtle bugs) if on a single line no brackets are necessary.